src/Analysis/Ast/Impl/Modules/PythonModule.cs

```diff
@@ -460,6 +460,11 @@ public override void Add(string message, SourceSpan span, int errorCode, Severit
                     OnAnalysisComplete();
                     ContentState = State.Analyzed;
 
+                    // Do not report issues with libraries or stubs
```

Comment:
What if user has opened library file explicitly?

Reply:
Also see: https://github.com/Microsoft/python-language-server/pull/584#discussion_r255287562

Reply:
We don’t report issues in libraries, this is not something actionable to the user. Something like resharper does not produce issues in disassembled code...